inc/invitation.class.php

```diff
@@ -78,6 +86,11 @@ public function getRights($interface = 'central') {
       return $rights;
    }
 
+   /**
+    * Prepare input datas for adding the item
+    * @param array $input
```

Comment:
$input is the data to use when creating a new row or updating an existing in the DB respectively for prepareInputForAdd() and prepareInputForUpdate().